projects/plugins/super-cache/changelog/super-cache-missing-action

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Hey @Automattic/jetpack-crew!
I didn't add those super-cache files.
I think this was a side-effect of the merges.
How should I proceed?

Reply:
I'd recommend removing them from your PR. We don't want to add them if they're not necessary.

Reply:
Weird, not sure what happened there. Doing some digging, it looks like Super Cache [included those changelogs](https://github.com/Automattic/jetpack/pull/28520/files) in its release the other day, so I would just delete them.